README.md

```diff
@@ -108,34 +108,30 @@ their `onRecord` streams.
   });
 
 
-  // Will NOT print because FINER is too low level for `Logger.root`.
+  // Will NOT print because FINER is too low level for ([LOG1] & [ROOT]).
   log1.finer('LOG_01 FINER (X)');
 
-  // Will print twice ([LOG1] & [ROOT])
-  log1.fine('LOG_01 FINE (√√)');
+  // Will print ONCE by [LOG1] because FINE is too low level for [ROOT].
+  log1.fine('LOG_01 FINE (√)');
 
-  // Will print ONCE because `log1` only uses root listener.
-  log1.warning('LOG_01 WARNING (√)');
```

Comment:
This one is incorrect.
It should print twice as `WARNING` is sufficient for both [LOG1] & [ROOT], and the output actually print it twice as:
```
[LOG1][FINE+] LOG_01 WARNING (√)
[ROOT][WARNING+] LOG_01 WARNING (√)
```